src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -4,16 +4,24 @@ import Model from './../../utils/model';
 import WebcamCapture from '../WebcamCapture/WebcamCapture';
 import Logo from '../Logo/Logo';
 import './ItemRecognition.css';
+import {ControllerDataset} from '../Admin/Trainer/ControllerDataset';
+import MobileNet from '../Admin/Trainer/MobileNet';
 
 const ML_THRESHOLD = 0.06;
 
 class ItemRecognition extends Component {
   model = new Model();
   webcam = React.createRef();
 
+  constructor(props) {
+    super(props);
+    this.mobilenet = new MobileNet();
+  }
```

Comment:
Does this need a constructor?

Reply:
Good spot! Can place it directly under webcam to be a bit less verbose.

Would `mobileNet` be more consistent since they're two words and that's how the class is named?